types/httpstatus.pp

```diff
@@ -1,2 +1,4 @@
 # @summary Validate a HTTP status code
-type Stdlib::HttpStatus = Integer[100, 599]
+# @deprecated Use Stdlib::Http::Status
+# @see Stdlib::Http::Status
+type Stdlib::HttpStatus = Stdlib::Http::Status
```

Comment:
after this is merged, can you raise a new PR that removes this old type? Then we can keep it open until the next major release and won't forget it.

Reply:
sure will do, ill leave this open for now so i dont forget